addons/modules/functions/fnc_moduleEditTerrainLocation.sqf

```diff
@@ -0,0 +1,108 @@
+#include "script_component.hpp"
+/*
+ * Author: Ampersand
+ * Zeus module function to edit terrain locations.
+ *
+ * Arguments:
+ * 0: Logic <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [LOGIC] call zen_modules_fnc_moduleEditTerrainLocation
+ *
+ * Public: No
+ */
+
+params ["_logic"];
+
+private _location = nearestLocation [_logic, ""];
+
+private _type = type _location;
+private _text = text _location;
+private _direction = direction _location;
+private _isRectangular = rectangular _location;
+private _size = size _location;
+private _name = name _location;
+private _side = side _location;
+private _importance = importance _location;
+
+private _types = ["Mount","Name","Strategic","StrongpointArea","FlatArea","FlatAreaCity","FlatAreaCitySmall","CityCenter","Airport","NameMarine","NameCityCapital","NameCity","NameVillage","NameLocal","Hill","ViewPoint","RockArea","BorderCrossing","VegetationBroadleaf","VegetationFir","VegetationPalm","VegetationVineyard","Area","Flag","Invisible","HistoricalSite","CivilDefense","CulturalProperty","DangerousForces","SafetyZone","HandDrawnCamp","b_unknown","o_unknown","n_unknown","b_inf","o_inf","n_inf","b_motor_inf","o_motor_inf","n_motor_inf","b_mech_inf","o_mech_inf","n_mech_inf","b_armor","o_armor","n_armor","b_recon","o_recon","n_recon","b_air","o_air","n_air","b_plane","o_plane","n_plane","b_uav","o_uav","n_uav","b_naval","o_naval","n_naval","b_med","o_med","n_med","b_art","o_art","n_art","b_mortar","o_mortar","n_mortar","b_hq","o_hq","n_hq","b_support","o_support","n_support","b_maint","o_maint","n_maint","b_service","o_service","n_service","b_installation","o_installation","n_installation","u_installation","b_antiair","o_antiair","n_antiair","c_unknown","c_car","c_ship","c_air","c_plane","group_0","group_1","group_2","group_3","group_4","group_5","group_6","group_7","group_8","group_9","group_10","group_11","respawn_unknown","respawn_inf","respawn_motor","respawn_armor","respawn_air","respawn_plane","respawn_naval","respawn_para"];
```

Comment:
Can we not dynamically create this list from `CfgLocationTypes`?

Reply:
Sure can! [cf30381](https://github.com/zen-mod/ZEN/pull/751/commits/cf3038196479d0b48f5922b825ed9d42d2e2e569)